packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/migrations/transform_set_managed_default.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SavedObjectUnsanitizedDoc } from '@kbn/core-saved-objects-server';
+
+export const transformSetManagedDefault = (
+  doc: SavedObjectUnsanitizedDoc<unknown> | undefined
```

Comment:
NIT: why this `| undefined` here? I this just because of the chaining via lodash or something? Ideally we would explicitly use the `TransformFn` type here as it was done for `transformMigrationVersion` in the same folder.

Reply:
- Yea, after changes, `| undefined` isn't needed. Thanks for pointing that out.
- We can't use `TransformFn` because `transformMigrationVersion` converts `doc` -> `{ transformedDoc, 
additionalDocs }`. That's why we need to `get` `transformedDoc` as an input to `transformSetManagedDefault`.